dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -439,6 +451,23 @@ public JdbcEventStore( StatementBuilder statementBuilder, JdbcTemplate jdbcTempl
         return events;
     }
 
+    private ProgramStageInstanceUserInfo jsonToUserInfo( String userInfoAsString )
```

Comment:
Move to `EventUtils` class as static method

Reply:
Done, thank you